frontend/src/i18n.js

```diff
@@ -6,9 +6,10 @@ import { defaultNS, resources } from "./locales/resources";
 i18next
   // https://github.com/i18next/i18next-browser-languageDetector/blob/9efebe6ca0271c3797bc09b84babf1ba2d9b4dbb/src/index.js#L11
   .use(initReactI18next) // Initialize i18n for React
-  .use(LanguageDetector)
+  .use(LanguageDetector) // Initialize i18n for React
   .init({
-    fallbackLng: "en",
+    lng: window.navigator.language || "ru",
```

Comment:
The default language should not be changed - can uncheck all changes from this file